helmfile.tpl/helmfile-autoconfig.yaml

```diff
@@ -0,0 +1,19 @@
+bases:
```

Comment:
could you please rename this file to `helmfile-e2e.yaml`

Reply:
Done